src/modules/jackrack/filter_lv2.c

```diff
@@ -0,0 +1,279 @@
+/*
+ * filter_ladspa.c -- filter audio through LADSPA plugins
+ * Copyright (C) 2004-2018 Meltytech, LLC
```

Comment:
Change year in copyright for all files. For modified files the range end year is 2024. For new files, the Meltytech copyright line should be only the year 2024.